.github/workflows/build.yml

```diff
@@ -427,6 +427,57 @@ jobs:
             library-x86_64-linux-android
           failOnError: false
 
+  alpha-release:
+    name: Alpha Release
+    if: github.event_name == 'push' && github.ref == 'refs/heads/main'
+    permissions:
+      id-token: write
+    runs-on: ubuntu-latest
+    env:
+      NPM_TAG: 'alpha'
+    needs: 
+      - create-ios-xcframework
+      - create-android-library
```

Comment:
This seems to just release the ios/android wrappers as alpha. Not the Node.JS.

I would really like to keep consistency between node.js and react native.

My proposal for this PR would be:
- on push to main, build the binaries for indy vdr
- update the version of the wrapper to the current package.json version but with e.g. .alpha-{commit-sha} appended
- create a prerelease on github and push the binaries
- release the JS wrapper

Reply:
The github pre releaase could e.g. be called "JS 0.2.2 Alpha {sha-commit}" to make it clear this is a JS pre-release.

Then the package.json binary is updated to link to this pre-release.

This way pre-releases work the same as normal releases.

Reply:
It seems messy to manage different lifecycle in the same repository. I think we need to move to [uniffi-rs](https://mozilla.github.io/uniffi-rs/latest/) or some other tool. I am concern about those wrappers getting out of sync and avoid missing release, and release from side branches, like we it was done.